### PR TITLE
The Great Indentation Fix of 2015

### PR DESCRIPTION
**Pull request by vassilevsky**
At first I deleted individual indentation cops. But that failed to fix indentation everywhere for some reason. So finally I just deleted all indentation-related cops and ran `rubocop -a`. This still failed to fix `when` indentation in two places. I did it manually (second commit).

This PR is for human checking purposes. GitHub does a good job of highlighting changes. There should be none! Only indentation fixes, without any changes to logic.

Please, have a look and let me know if everything is alright. I will check it too of course.

### CODE REVIEW

File: doc/ex/InitialCoords.rb [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
-        grp.rect(3, 3, 297,  0)
-        grp.rect(3, 3,   0, 97)
-    end
-    canvas.g.styles(:font_size=>14, :font_family=>'Verdana', :font_weight=>'normal', :font_style=>'normal') do |grp|
-        grp.text(10, 20, '(0,0)')
-        grp.text(240, 20, '(300,0)')
-        grp.text(10, 90, '(0,100)')
-    end
+  canvas.desc = "Example InitialCoords - SVG's initial coordinate system"
+  canvas.background_fill = 'white'
```
Comment (vassilevsky) [inline doc/ex/InitialCoords.rb]: Stupid dog, I already included `.rubocop.yml` in `.hound.yml`, what else do you need?

File: doc/ex/arcs01.rb [hunk 2]
```diff
@@ -3,26 +3,26 @@
...
+  canvas.background_fill = 'white'
+  canvas.rect(1196, 395, 1, 1).styles(:fill=>'none', :stroke=>'blue', :stroke_width=>1)
 
-    canvas.path('M300,200 h-150 a150,150 0 1,0 150,-150 z')
-                .styles(:fill=>'red', :stroke=>'blue', :stroke_width=>5)
-    canvas.path('M275,175 v-150 a150,150 0 0,0 -150,150 z')
-                .styles(:fill=>'yellow', :stroke=>'blue', :stroke_width=>5)
-    canvas.path(<<-END_PATH
+  canvas.path('M300,200 h-150 a150,150 0 1,0 150,-150 z')
+  .styles(:fill=>'red', :stroke=>'blue', :stroke_width=>5)
```
Comment (vassilevsky) [inline doc/ex/arcs01.rb]: I think it's necessary to indent the chained call.

File: doc/ex/colors.rb [hunk 3]
```diff
@@ -15,16 +15,16 @@
...
-        rgb  = sprintf('#%02x%02x%02x', c.color.red&0xff,  c.color.green&0xff, c.color.blue&0xff)
-        rgb += sprintf('%02x', c.color.opacity&0xff) if c.color.opacity != 0
-        m = /(.*?)Compliance/.match c.compliance.to_s
-        colors.cur_image['Label'] = "#{c.name} (#{rgb}) #{m[1]}"
+  if c.name !~ /grey/    # omit SVG 'grays'
+    colors.new_image(200, 25) do
+      self.background_color = c.color
+      self.border_color = 'gray50'
     end
+    rgb  = sprintf('#%02x%02x%02x', c.color.red&0xff,  c.color.green&0xff, c.color.blue&0xff)
```
Comment (vassilevsky) [inline doc/ex/colors.rb]: It is necessary for equal signs alignment.

**Comment by linduxed** [comment]
Since you changed the `.hound.yml` file to prevent Hound from complaining about the wrong things, please delete the Hound comments which are no longer conforming with the new `.hound.yml`.

Other than that, the PR indeed doesn't have anything else than whitespace fixes. Should be good to merge.

**Comment by mockdeep** [comment]
How come we're building on semaphore now, too?

**Comment by vassilevsky** [comment]
I tried building other stuff on Semaphore and it was really fast. So I decided to try to employ it for RMagick.

The biggest obstacle so far is that it doesn't support matrices. At the moment I don't realize how to visualize all Ruby and ImageMagick version combinations on Semaphore.

I'm gonna turn it off for now then.

**Comment by vassilevsky** [comment]
https://github.com/gemhome/rmagick/pull/174/files?w=1

Adding `?w=1` to GitHub diff URLs is equivalent to running `git diff -w` which shows only real code changes, **not** whitespace changes. It kinda proves that there are only whitespace changes in this branch. Travis build also passed. So, I'm merging this.

**Pull request merged**